cmd/call-endpoint/main_test.go

```diff
@@ -40,6 +41,56 @@ func TestCallApiEndpointBase(test *testing.T) {
 			},
 		},
 
+		// Custom Output Formatters.
+		// Single key with a list.
+		{
+			CommonCMDTestCase: cmd.CommonCMDTestCase{
+				ExpectedStdout: EXPECTED_COURSES_ASSIGNMENTS_TABLE,
+			},
+			endpoint: "courses/assignments/list",
+			parameters: []string{
+				"course-id:course101",
+				"--table",
+			},
+		},
+		// Single key with json.
+		{
+			CommonCMDTestCase: cmd.CommonCMDTestCase{
+				ExpectedStdout: EXPECTED_COURSES_ASSIGNMENTS_TABLE,
```

Comment:
Same output?

Assuming they have the same output,
you should either generalize the variable name or alias it by creating a new variable that is initialized with the other variable.